package.json

```diff
@@ -15,7 +15,7 @@
     "start": "cozy-scripts start --hot",
     "deploy": "cs deploy",
     "cozyPublish": "cozy-scripts publish --token $REGISTRY_TOKEN --prepublish downcloud --postpublish mattermost",
-    "test": "cozy-scripts test",
+    "test": "cozy-scripts test --env=jest-environment-jsdom-sixteen",
```

Comment:
Note that this wouldn't be needed with a newer version of jest in create-cozy-app. Would it be a thing to upgrade it in our projects?

https://github.com/testing-library/dom-testing-library/releases/tag/v7.0.0

Reply:
https://github.com/cozy/create-cozy-app/issues/1419